benchmark/c_str-functions/c_str-impl.h

```diff
@@ -0,0 +1,24 @@
+#include "zoo/swar/SWAR.h"
+
+/// \brief Best implementation of Lemire's technique
+///
+/// 128 integers are currently an emulation that uses 64 computation
```

Comment:
'128 bit integers are currently an emulation that uses 64 bit integer hardware' reads better